envs/.env.dev

```diff
@@ -6,6 +6,7 @@ NEXT_IMAGE_DOMAIN=https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.
 # NEXT_PUBLIC_ASSETS_URL=https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/
 NEXT_PUBLIC_ASSETS_URL=/generated/
 SITE_URL=https://www.va.gov
+NEXT_PUBLIC_SITE_URL=https://www.va.gov
```

Comment:
Is there a reason we're not using dev.va.gov here?

Reply:
I was just copying prod and maybe up a bit late. Will update

Reply:
Updated